Core/GDCore/Extensions/Metadata/ParameterMetadataTools.cpp

```diff
@@ -71,7 +75,7 @@ void ParameterMetadataTools::ParametersToObjectsContainer(
           const gd::String& behaviorType = parameter.GetExtraInfo();
 
           gd::Object& object = outputObjectsContainer.GetObject(lastObjectName);
-          allObjectBehaviorNames[lastObjectName].insert(behaviorName);
+          allObjectNonDefaultBehaviorNames[lastObjectName].insert(behaviorName);
```

Comment:
I wonder if the name can be confusing because a free function could have an object parameter without object type and the following behavior parameters be some capabilities. It also won't be "all" the behaviors of the actual objects.
I guess it's kind of the required behavior for objects passed in parameter, but "required" may be confusing as it's already used for behavior properties.

```suggestion
          objectAdditionnalBehaviorNames[lastObjectName].insert(behaviorName);
```